src/yuzu/main.cpp

```diff
@@ -3381,36 +3381,38 @@ void GMainWindow::filterBarSetChecked(bool state) {
 }
 
 void GMainWindow::UpdateUITheme() {
-    const QString default_icons = QStringLiteral("default");
-    const QString& current_theme = UISettings::values.theme;
-    const bool is_default_theme = current_theme == QString::fromUtf8(UISettings::themes[0].second);
+    const QString default_theme = QStringLiteral("default");
+    QString current_theme = UISettings::values.theme;
     QStringList theme_paths(default_theme_paths);
 
-    if (is_default_theme || current_theme.isEmpty()) {
-        const QString theme_uri(QStringLiteral(":default/style.qss"));
+    if (current_theme.isEmpty()) {
+        current_theme = default_theme;
+    }
+
+    if (current_theme != default_theme) {
+        QString theme_uri(QLatin1Char{':'} + current_theme + QStringLiteral("/style.qss"));
```

Comment:
```suggestion
        QString theme_uri{QStringLiteral(":%1/style.qss").arg(current_theme)};
```

Seems like it would be a little more concise

Reply:
Done!